make/data/fontconfig/windows.fontconfig.properties

```diff
@@ -144,56 +145,56 @@ dialog.plain.chinese-ms950=MingLiU
 dialog.plain.chinese-ms950-extb=MingLiU-ExtB
 dialog.plain.hebrew=David
 dialog.plain.japanese=MS Gothic
-dialog.plain.korean=Gulim
+dialog.plain.korean=Malgun Gothic
 
 dialog.bold.alphabetic=Arial Bold
 dialog.bold.chinese-ms950=PMingLiU
 dialog.bold.chinese-ms950-extb=PMingLiU-ExtB
 dialog.bold.hebrew=David Bold
 dialog.bold.japanese=MS Gothic
-dialog.bold.korean=Gulim
+dialog.bold.korean=Malgun Gothic Bold
 
 dialog.italic.alphabetic=Arial Italic
 dialog.italic.chinese-ms950=PMingLiU
 dialog.italic.chinese-ms950-extb=PMingLiU-ExtB
 dialog.italic.hebrew=David
 dialog.italic.japanese=MS Gothic
-dialog.italic.korean=Gulim
+dialog.italic.korean=Malgun Gothic
 
 dialog.bolditalic.alphabetic=Arial Bold Italic
 dialog.bolditalic.chinese-ms950=PMingLiU
 dialog.bolditalic.chinese-ms950-extb=PMingLiU-ExtB
 dialog.bolditalic.hebrew=David Bold
 dialog.bolditalic.japanese=MS Gothic
-dialog.bolditalic.korean=Gulim
+dialog.bolditalic.korean=Malgun Gothic
 
 dialoginput.plain.alphabetic=Courier New
 dialoginput.plain.chinese-ms950=MingLiU
 dialoginput.plain.chinese-ms950-extb=MingLiU-ExtB
 dialoginput.plain.hebrew=David
 dialoginput.plain.japanese=MS Gothic
-dialoginput.plain.korean=Gulim
+dialoginput.plain.korean=Malgun Gothic
 
 dialoginput.bold.alphabetic=Courier New Bold
 dialoginput.bold.chinese-ms950=PMingLiU
 dialoginput.bold.chinese-ms950-extb=PMingLiU-ExtB
 dialoginput.bold.hebrew=David Bold
 dialoginput.bold.japanese=MS Gothic
-dialoginput.bold.korean=Gulim
+dialoginput.bold.korean=Malgun Gothic Bold
 
```

Comment:
Since dialoginput is usually the same as monospaced, per Naoto's comment, this should be reverted.

Reply:
Not only reverting, but I think `GulimChe` here is the right choice, as it is `monospaced`?